### PR TITLE
Use HTTPS

### PR DESCRIPTION
**Pull request by carusogabriel**
Use HTTPS when possible, and also simplifying `https://secure.php.net` to just `https://php.net` :+1:

### CODE REVIEW

**Comment by SpacePossum** [comment]
good idea, thanks :)

File: doc/schema.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$schema": "https://json-schema.org/draft-04/schema#",
```
Comment (julienfalque) [inline doc/schema.json]: Looks like the certificate for this one is not valid, I get a "This connection is not secure" page when trying to reach it.

File: tests/Fixer/Phpdoc/PhpdocInlineTagFixerTest.php [hunk 2]
```diff
@@ -153,7 +153,7 @@ public function provideTestFixInlineDocCases()
         $cases[] = array(
             '<?php
     /**
-     * {@link http://www.ietf.org/rfc/rfc1035.text)
+     * {@link https://www.ietf.org/rfc/rfc1035.text)
```
Comment (julienfalque) [inline tests/Fixer/Phpdoc/PhpdocInlineTagFixerTest.php]: This one is a 404, it should be `.txt`, not `.text`.

**Comment by keradus** [comment]
Thank you @carusogabriel.

**Pull request merged**